pkg/csi/service/common/authmanager.go

```diff
@@ -567,6 +580,9 @@ func getFSEnabledClustersWithPriv(ctx context.Context, vc *cnsvsphere.VirtualCen
 			config.FileServiceConfig.Enabled)
 		if config.FileServiceConfig.Enabled {
 			clusterComputeResourceWithPrivAndFS = append(clusterComputeResourceWithPrivAndFS, cluster)
+		} else {
```

Comment:
Would it be better to remove the log at line 579, add an Info log in the `if` block and a Debug log in the `else` block?

Reply:
Looks like this function is called on every CreateVolume call, so just optimizing what we log

Reply:
I was thinking that we are interested in case where vSAN file service is NOT enabled. If someone has not enabled vSAN file service and tries to create file volume, then based on this log we can ask user to enable vSAN file service.
So is it ok if we move debug log in `if` block and keep info log in `else` block as it is?